Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -753,20 +753,6 @@ BarProp& LocalDOMWindow::toolbar()
     return *m_toolbar;
 }
 
-PageConsoleClient* LocalDOMWindow::console() const
-{
-    // FIXME: This should not return nullptr when frameless.
-    if (!isCurrentlyDisplayedInFrame())
-        return nullptr;
```

Comment:
Doesn't look like this part was moved, is it ok to drop this?

Reply:
Yes.